NitroxClient/MonoBehaviours/NitroxDebugManager.cs

```diff
@@ -125,5 +126,34 @@ private void ShowDebuggers()
             }
             prevActiveDebuggers.Clear();
         }
+
+        private void OnEnable()
+        {
+            SceneManager.sceneLoaded += SceneManager_sceneLoaded;
+            SceneManager.sceneUnloaded += SceneManager_sceneUnloaded;
+            SceneManager.activeSceneChanged += SceneManager_activeSceneChanged;
+        }
+
+        private void OnDisable()
+        {
+            SceneManager.sceneLoaded -= SceneManager_sceneLoaded;
+            SceneManager.sceneUnloaded -= SceneManager_sceneUnloaded;
+            SceneManager.activeSceneChanged -= SceneManager_activeSceneChanged;
+        }
+
+        private void SceneManager_sceneLoaded(Scene scene, LoadSceneMode loadMode)
```

Comment:
Was this for your own debugging purposes, or do you intend to leave it in here for future adventures?

Reply:
It is purely for getting more technical information in the output-log.txt. It's easier to see where an error occurred in the game if you know what got loaded etc.

Reply:
Right, sounds like a helpful thing to keep in there.